src/charm.py

```diff
@@ -464,7 +481,7 @@ def _config_changed(self, event: HookEvent) -> None:
             self._config_force_https()
             self.model.unit.status = ActiveStatus()
 
-    def _reload_configuration(self) -> None:
+    def _reload_configuration(self, _=None) -> None:
```

Comment:
I'm guessing this is because this is now used as an event handler, as noted above each event should have its own dedicated event handling function

Reply:
Removed with the event handler renaming